src/marqo/tensor_search/api.py

```diff
@@ -515,101 +566,30 @@ def rollback_vespa_app_to_current_version(marqo_config: config.Config = Depends(
     )
 
 
+@app.post('/validate/index/{index_name}', include_in_schema=False)
+@utils.enable_ops_api()
+def schema_validation(index_name: str, settings_object: dict):
+    IndexManagement.validate_index_settings(index_name, settings_object)
+
+    return JSONResponse(
+        content={
+            "validated": True,
+            "index": index_name
+        }
+    )
+
+
+@app.get("/health" , include_in_schema=False)
+def check_health(marqo_config: config.Config = Depends(get_config)):
+    health_status = marqo_config.monitoring.get_health()
+    return HealthResponse.from_marqo_health_status(health_status)
+
+
+@app.get('/memory', include_in_schema=False)
+@utils.enable_debug_apis()
+def memory():
+    return memory_profiler.get_memory_profile()
+
+
 if __name__ == "__main__":
     uvicorn.run(app, host="localhost", port=8882)
-
```

Comment:
These documents are outdated, and we have similar samples in Marqodocs. So I removed them from here.